config.yaml

```diff
@@ -58,3 +58,7 @@ options:
   webserver_workers:
     type: int
     description: The number of webserver worker processes for handling requests.
+  webserver_wsgi_path:
```

Comment:
I'm wondering if we shouldn't name this `webserver_entrypoint` as it's more generic/reusable and path could look like FS path

Reply:
WSGI path is a common technical term in the Flask domain, so I think it might be more understandable for Flask users.

Reply:
Yeap but we're in the webserver module, which is meant to become generic

Reply:
I think the WSGI path is applicable for all Flask web servers, such as Gunicorn, uWSGI, etc.

Reply:
The problem is that wsgi path is specify to the WSGI server, for other type of web servers such as fastcgi server, there's a correlating concept to the wsgi path in WSGI server. Maybe we can rename the `webserver` module to `wsgi_webserver` module?

Reply:
For other configurations, there are similar problems, for example, a cgi webserver doesn't have the concept of workers or threads.

Reply:
The renaming later sounds like a good option when we have another example, e.g., the fastAPI or sprint boot case